trade_remedies_api/organisations/services/v2/serializers.py

```diff
@@ -96,6 +97,14 @@ def to_internal_value(self, data):
             data["country"] = data["country"]["alpha3"]
         return data
 
+    @staticmethod
+    def get_potential_duplicate_organisations(instance):
+        """get a serialize-able duplicate organisation list"""
+        return [
+            duplicate_org.to_embedded_dict()
```

Comment:
we should not be using .to_embedded_dict() in V2 code, as it is V1 code.

```suggestion
            OrganisationSerializer(instance=duplicate_org).data
```